### PR TITLE
Feature/parquet readstep

### PR DESCRIPTION
**Pull request by zaychenko-sergei**

### CODE REVIEW

File: schemas/fragments/ReadStep.json [hunk 1]
```diff
@@ -158,6 +158,21 @@
           "description": "Path to a data file within a multi-file archive. Can contain glob patterns."
         }
       }
+    },
+    "Parquet": {
+      "description": "Reader for ESRI Shapefile format.",
```
Comment (sergiimk) [inline schemas/fragments/ReadStep.json:163]: Description needs to be updated

File: .gitignore [hunk 2]
```diff
@@ -1 +1,2 @@
 /build
+/tools/utils/__pycache__
```
Comment (sergiimk) [inline .gitignore:2]: My bad, thanks for the cleanup!

Might be better to specify just `__pycache__` (without the path) so it applies to all directories.

**Review by sergiimk — APPROVED** [review]
Few minor things and will be good to merge.

Please also bump the minor version in:
- `README` (inside the `Latest Version` badge text that goes like `Spec-0.25.0-green`)
- `src/open-data-fabric.md` at the top (and don't forget to run `make` again)

Also add an entry in the `CHANGELOG` and don't forget to tag as `v0.26.0` once you merge.

... I know it's a lot - we should automate all these things.

File: schemas/fragments/ReadStep.json (hunk 1 above)
Comment (zaychenko-sergei) [reply schemas/fragments/ReadStep.json]: My bad

File: .gitignore (hunk 2 above)
Comment (zaychenko-sergei) [reply .gitignore]: Good point

**Comment by zaychenko-sergei** [comment]
> Few minor things and will be good to merge.
> 
> Please also bump the minor version in:
> 
> * `README` (inside the `Latest Version` badge text that goes like `Spec-0.25.0-green`)
> * `src/open-data-fabric.md` at the top (and don't forget to run `make` again)
> 
> Also add an entry in the `CHANGELOG` and don't forget to tag as `v0.26.0` once you merge.
> 
> ... I know it's a lot - we should automate all these things.

Updated

**Pull request merged**